src/OpenTelemetry.Api/Context/Propagation/ITextFormat.cs

```diff
@@ -45,10 +45,11 @@ public interface ITextFormat
         /// Extracts activity context from textual representation.
         /// </summary>
         /// <typeparam name="T">Type of object to extract context from. Typically HttpRequest or similar.</typeparam>
+        /// <param name="activityContext">Activity context to extract from the wire.</param>
```

Comment:
this comment is incorrect, isn't it?

Reply:
it is. but i don't know how would be the better words to replace. Do you have any suggestion?

Reply:
How about: "The default activity context to be used if Extract fails."